pkg/apis/devconsole/v1alpha1/component_types.go

```diff
@@ -36,8 +36,39 @@ type Component struct {
 	Status ComponentStatus `json:"status,omitempty"`
 }
 
-func (c *Component) GetName() string {
-	return c.Name
+// GetLabelName retrieves the name label. If not present,
```

Comment:
[SUGGESTION] Shouldn't this be GetLabelName retrieves the _label name_

Reply:
@DhritiShikhar perhaps it's more clear the way you have it :-) will update!

Reply:
done! 57ac5972a39628ccde40588f6e0a55c63f750790